api/include/opentelemetry/trace/span_id.h

```diff
@@ -66,6 +67,16 @@ class SpanId final
     memcpy(dest.data(), rep_, kSize);
   }
 
+  static SpanId GetRandom()
+  {
+    uint8_t buf[kSize] = {0};
+    for (int i = 0; i < kSize; i++)
+    {
+      buf[i] = rand() % 256;
```

Comment:
I don't think we should use rand()

Reply:
I need a way to randomly generate numbers and I don't know how else to do it in api. And considering that the random number is used within the local function it should be safe?